examples/webpack.config.js

```diff
@@ -38,6 +44,8 @@ module.exports = {
   },
   plugins: [
     plugins.html({ title: 'React Big Calendar' }),
-    plugins.extractCss(),
+    plugins.extractCss({
+      filename: 'static/[name].[hash].css',
```

Comment:
also shouldn't be needed, things will be output to the the `output.path` automatically